### PR TITLE
Disable finish gitops setup button onclick as well and not just the s…

### PR DESCRIPTION
**Pull request by salahalsaleh1993**
…tyle

### CODE REVIEW

File: web/src/components/gitops/GitOpsRepoDetails.jsx [hunk 1]
```diff
@@ -270,7 +270,13 @@ class GitOpsRepoDetails extends React.Component {
 
             <div className="flex">
               {this.props.showCancelBtn && <button className="btn secondary dustyGray u-marginRight--10" type="button" onClick={this.props.onCancel}>Cancel</button>}
-              <button className={`btn primary blue ${(finishingSetup || !this.allowUpdate()) && "is-disabled"}`} type="button" onClick={this.onFinishSetup}>{finishingSetup ? "Finishing setup" : "Finish GitOps setup"}</button>
+              <button
+                className={`btn primary blue ${(finishingSetup || !this.allowUpdate()) && "is-disabled"}`}
+                type="button"
+                onClick={(finishingSetup || !this.allowUpdate()) ? null : this.onFinishSetup}
```
Comment (GraysonNull) [inline web/src/components/gitops/GitOpsRepoDetails.jsx]: is there a reason we don't just add this attribute? `disabled={finishingSetup || !this.allowUpdate()}`

Comment (salahalsaleh1993) [reply web/src/components/gitops/GitOpsRepoDetails.jsx]: oh my bad. nope, no reason, updating!

Comment (GraysonNull) [reply web/src/components/gitops/GitOpsRepoDetails.jsx]: 👍 thanks

**Comment by salahalsaleh1993** [comment]
the `disabled` prop does both things

**Pull request merged**